functions/backup.bash

```diff
@@ -483,15 +488,15 @@ EOF
   partprobe
   cond_redirect mke2fs -F -t ext4 "${dest}3"
   mkdir -p "${storageDir}"
-  if ! sed -e "s|%DEVICE|${backupdrive}3|g" -e "s|%BKPDIR|${storageDir}|g" "${BASEDIR:-/opt/openhabian}"/includes/storage.mount >${targetDir}/storage.mount; then echo "FAILED (create storage mount)"; fi
+  if ! sed -e "s|%DEVICE|${backupdrive}3|g" -e "s|%BKPDIR|${storageDir}|g" "${BASEDIR:-/opt/openhabian}"/includes/storage.mount >${serviceTargetDir}/storage.mount; then echo "FAILED (create storage mount)"; fi
```

Comment:
Add a space between the output redirector and the file location, `> ${serviceTargetDir}/storage.mount`

Reply:
ok